storage-checks.sh

```diff
@@ -0,0 +1,374 @@
+#!/bin/bash
+
+: '
+This scipt will validate blob storage from AIC perspective.
+# $1 - json file with credentials
+[Script Version -> 21.4]
+Credentials file structure
+{"AWS_HOST": <AWS_HOST>, "AWS_ENDPOINT": <AWS_ENDPOINT>, "AWS_ACCESS_KEY_ID": <AWS_ACCESS_KEY_ID>, "AWS_SECRET_ACCESS_KEY": <AWS_SECRET_ACCESS_KEY>, "BUCKET_1": <BUCKET_1>, "BUCKET_2": <BUCKET_2>}
+where the access_key and secret correspond to aws crednetials which has access to the two buckets (customer creates buckets with appropriate policies) 
+Requirements: aws s3, jq
+'
```

Comment:
Lets include help section in case any of the input section or command parameters are not valid.